src/AlertsManagement/AlertsManagement/ChangeLog.md

```diff
@@ -0,0 +1,24 @@
+<!--
```

Comment:
It looks like the psd1 file above has some strange encoding now.  Can you make sure this is ascii/utf-8?